policy/modules/system/systemd.if

```diff
@@ -2256,3 +2256,80 @@ interface(`systemd_delete_private_tmp',`
 	allow $1 systemd_private_tmp_type:lnk_file delete_lnk_file_perms;
 	allow $1 systemd_private_tmp_type:sock_file delete_sock_file_perms;
 ')
+
+#######################################
+## <summary>
+##	Create objects in the pid directory
+##	with a private type with a type transition.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`systemd_userdbd_data_pid_filetrans',`
+	gen_require(`
+		type init_var_run_t;
+		type systemd_userdbd_data_t;
+	')
+
+	filetrans_pattern($1, init_var_run_t, systemd_userdbd_data_t, dir, "userdb")
```

Comment:
I would not call this "data", instead "runtime" seems more appropriate

Reply:
@doverride , yes I also think runtime is more appropriate. Will change it.